src/js/components/AppPageComponent.jsx

```diff
@@ -439,12 +440,14 @@ var AppPageComponent = React.createClass({
       content = this.getTaskDetailComponent();
     }
 
+    var group = ViewHelper.getGroupFromAppId(state.appId);
+
     return (
       <div>
-        <AppBreadcrumbsComponent
-          activeTaskId={state.activeTaskId}
-          activeViewIndex={state.activeViewIndex}
-          appId={state.appId} />
+        <BreadcrumbComponent
+          group={group}
+          app={state.appId}
```

Comment:
Yeah, I was unsure so decided to save on characters but I think that's a good clarification.